### ISSUE TITLE
Exercise 3.7.3-4 DFA wrong diagram

### ISSUE THREAD

**Issue by Michael137** [fool2fish/dragon-book-exercise-answers#140]
Should have an edge labelled "b" from I to G.

On a side-note, why do we have the b-edge from G going back to G instead of to C? When in state G, an input of "b" would yield: e-closure({5}), which is C

**Comment by elenidriva** [comment]
Correct, there are two missing transitions. From C to C with a label "b" and from I to G with a label "b" again.
